src/lib/classes/styling/ZodSchemas/MediaSchemes.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+import MediaScheme from "./MediaScheme";
+
+const MediaSchemes = z.object({
+	default: MediaScheme,
```

Comment:
```suggestion
	default: MediaScheme.required(),
```
All members in the default schema should be defined, that way we are sure that no variable is undefined.

NOTE: I am not sure if `required` is a nested modifier or only shallow. We need it to be nested because we need all the individual variables to be required. Maybe you will need to make some more changes to fix that.

Docs:
https://zod.dev/?id=partial
https://zod.dev/?id=required